src/ASM/ASMs2Dmx.h

```diff
@@ -101,8 +101,9 @@ class ASMs2Dmx : public ASMs2D, private ASMmxBase
   //! \param neighbor The neighbor patch
   //! \param[in] nedge Local edge index of neighbor patch, in range [1,4]
   //! \param[in] revers Indicates whether the two edges have opposite directions
+  //! \param[in] coordCheck False to disable coordinate checks (periodic connections)
   virtual bool connectPatch(int edge, ASMs2D& neighbor, int nedge,
-			    bool revers = false);
+                            bool revers = false, bool coordCheck=true);
```

Comment:
Veldig kosmetisk: ekstra space rundt "="-tegn som for de andre initialisatorene (gjelder flere steder i samme patch).